src/core/cpuCO2.py

```diff
@@ -28,11 +29,39 @@ def cpu_co2(args = None, country_code = None) -> None:
         print("- Selected Country: {0} \n- Carbon Intensity: {1} ({2}) \n- Percentage of fossil fuel: {3}%".format(country_code, carbon_intensity, carbon_intensity_unit, fossil_percentage))
 
         # Retrieve cpu info
-        print('\nRetrieving CPU data...')
+        cpu_retrieval_time = 10 #to export in a configuration file
+        print(f'\nRetrieving CPU usage of the next {cpu_retrieval_time} seconds...')
         cpu_info = em.get_cpu_info()
-        cpu_usage = em.get_cpu_usage(10) #it seems too low to be true!
+        cpu_usage = em.get_cpu_usage(cpu_retrieval_time) #it seems too low to be true!
         print("- Current CPU: {0} \n- CPU usage during last 10s: {1}%".format(cpu_info, cpu_usage))
+
+        # Combine Co2 with CPU usage
+        print('\nCombining Co2 and CPU data...')
+        co2_emissions = combine_cpu_CO2(cpu_usage, cpu.get_cpu_tdp(), carbon_intensity)
+        print("- In the last {1} seconds your CPU footprint was: {0} g".format(co2_emissions, cpu_retrieval_time))
+
```

Comment:
Very minor thing but why not swapping the order of ```co2_emissions``` and ```cpu_retrieval_time``` to match the order in which they appear in the string? Basically just: ```.format(cpu_retrieval_time, co2_emissions)```.

Reply:
makes sense, corrected